### PR TITLE
Minor refactoring

### PR DESCRIPTION
**Pull request by stepango**

### CODE REVIEW

File: build-dependencies/dependencies/src/main/kotlin/ViewBinding.kt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 object viewbinding {
     val viewBinding = transitiveDeps(
-        "androidx.databinding:viewbinding:${Forma.configuration.agpVersion}"
+        "androidx.databinding:viewbinding"
```
Comment (grine4ka) [inline build-dependencies/dependencies/src/main/kotlin/ViewBinding.kt:3]: Why did you remove the version?

**Review by michaem — APPROVED** [review]
